src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdTypeDataProvider.java

```diff
@@ -9,101 +9,11 @@
   private AdTypeDataProvider() {
   }
 
-  @DataProvider
-  public static final Object[][] asyncSuccessWithAd() {
-    return new Object[][]{
-        {
-            "project43", "SyntheticTests/AdType/Async/Success",
-            "wka.life/_project43//article",
-            AdsContent.MOBILE_PREFOOTER,
-            "googlesyndication.com/pagead/imgad?id=CICAgKCNj62dEhCsAhj6ASgBMgjBw3U0lR5Thg"
-            // https://www.google.com/dfp/5441#delivery/LineItemDetail/lineItemId=260851332&orderId=245575332
-        },
-    };
-  }
-
-  @DataProvider
-  public static final Object[][] asyncHopNoAd() {
```

Comment:
What are the reasons we decide on removing this test case?

Reply:
Tests isn't removed, only provider 
https://github.com/Wikia/selenium-tests/pull/2195/files#diff-788533f3604f2f7912f879dcea9e412fR24